yt_dlp/extractor/fujitv.py

```diff
@@ -16,21 +16,32 @@ class FujiTVFODPlus7IE(InfoExtractor):
     }
 
     _TESTS = [{
-        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
+        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810076',
         'info_dict': {
-            'id': '5d40810075',
-            'title': '5d40810075',
+            'id': '5d40810076',
             'ext': 'mp4',
-            'format_id': '4000',
-            'thumbnail': 'http://i.fod.fujitv.co.jp/pc/image/wbtn/wbtn_5d40810075.jpg'
+            'title': '#1318 『まる子、まぼろしの洋館を見る』の巻',
+            'series': 'ちびまる子ちゃん',
+            'series_id': '5d40',
+            'description': 'md5:b3f51dbfdda162ac4f789e0ff4d65750',
+            'thumbnail': 'http://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40810076_a.jpg',
         },
-        'skip': 'Expires after a week'
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        series_id = self._match_valid_url(url).group('sid')
+        self._download_webpage(url, video_id)
+        # TODO: extract thumbnail from webpage
+        # then remove trim part using regexp: \/(imf\/.*)\/img
+        # where the thumbnail is a img element with id of 'thumbnail'
         formats = self._extract_m3u8_formats(
             self._BASE_URL + 'abr/tv_android/%s.m3u8' % video_id, video_id, 'mp4')
+        if not self._get_cookies(url).get('CT') is None:
+            token = self._get_cookies(url).get('CT').value
+            json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id, video_id, headers={'x-authorization': f'Bearer {token}'}, fatal=False)
+        else:
+            print('token fetch failed')
```

Comment:
don't print anything.

Reply:
because the token is fetched successfully

Reply:
just for in case of the cookie method failed

Reply:
user doesn't need to know this,
all user cares about is downloading the video.

Reply:
imo adding a warning like this is correct, since it would align with how metadata extraction warns when non-fatally failing
More importantly, replace `print` with eiither `self.write_debug` or `self._report_warning`

Reply:
will do that tomo

Reply:
`token fetch failed` doesn't tell anything meaningful to the user.
Rather than printing this, it's better to add values from the webpage as fallback to all keys that are dependent on `json_info`